frontend/src/libs/types/burger-menu.type.ts

```diff
@@ -0,0 +1,12 @@
+import { type IconProp } from '@fortawesome/fontawesome-svg-core';
+
+import { type MenuLable } from '../enums/enums.js';
+import { type ValueOf } from './types.js';
+
+type MenuItem = {
+  label: ValueOf<typeof MenuLable>;
+  onClick: () => void;
+  icon: IconProp;
```

Comment:
If you use an `Icon` component in your code, you can use the `ValueOf<typeof IconName>` type here and not import `IconProp` from the library.

Reply:
It's also fixed, it just wasn't updated